ci-operator/step-registry/hypershift/kubevirt/install/hypershift/hypershift-kubevirt-install-hypershift-commands.sh

```diff
@@ -17,4 +17,5 @@ OPERATOR_IMAGE=${HYPERSHIFT_RELEASE_LATEST}
 bin/hypershift install --hypershift-image="${OPERATOR_IMAGE}" \
 --platform-monitoring=All \
 --enable-ci-debug-output \
---wait-until-available
+--wait-until-available \
+--enable-validating-webhook=true
```

Comment:
We should pass the evn var here

Reply:
I don't think so. In this case we want the webhook, because it's kubevirt related installation.